packages/jupyter-ai/jupyter_ai/tests/completions/test_handlers.py

```diff
@@ -27,9 +27,15 @@ def __init__(self, **kwargs):
             kwargs["responses"] = ["Test response"]
         super().__init__(**kwargs)
 
+    async def _acall(self, *args, **kwargs):
+        if hasattr(self, "raise_exc") and self.raise_exc:
+            raise Exception("Test exception")
+        else:
+            return super()._call(*args, **kwargs)
```

Comment:
Maybe `_acall` is no longer called with langchain v0.3?

Reply:
Found the error, see latest comment.